managed/src/main/java/com/yugabyte/yw/models/HealthCheck.java

```diff
@@ -36,15 +40,18 @@
 
   // The Json serialized version of the details. This is used only in read from and writing to the
   // DB.
+  @DbJson
   @Constraints.Required
   @Column(columnDefinition = "TEXT", nullable = false)
-  public String detailsJson;
+  public  Map<String, Object>  detailsJson;
```

Comment:
I checked and this is not a Map. It is setting from JsonData returned by healthCheck script. 
The way to deal with this is to create a POJO that captures the format of what the healthcheck script can return.  
i.e. Its going to be something like 
```
class HealthCheckDetails {
    boolean has_error;
    ... // other things 
}
```

May be all the properties are known and we can come up with such pojo sinc ethe health check script is in our control. So just check what format it generates.

If it is unbounded i.e. can contain any fields then we can just leave it as is.